test/fast/lfs-test.ts

```diff
@@ -11,7 +11,7 @@ describe('lfs', () => {
     const testRepoPath = temp.mkdirSync('desktop-git-lfs')
     const result = await GitProcess.exec([ 'lfs' ], testRepoPath)
     expect(result.exitCode).to.equal(0)
-    expect(result.stderr).to.contain('Git LFS is a system for managing and versioning large files')
+    expect(result.stdout).to.contain('Git LFS is a system for managing and versioning large files')
```

Comment:
I think this is something that was actually fixed in Git LFS: https://github.com/git-lfs/git-lfs/pull/2210